src/Runner.Worker/ExecutionContext.cs

```diff
@@ -741,6 +741,9 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
             // File table
             Global.FileTable = new List<String>(message.FileTable ?? new string[0]);
 
+            // What type of job request is running (i.e. Run Service vs. pipelines)
+            Global.Variables.Set("job_request_type", message.MessageType);
```

Comment:
put `job_request_type` into the constants file.
also, name it `system.jobRequestType`.